interface/forms/vitals/vitals.js

```diff
@@ -77,27 +83,65 @@
                     inputSave.value = value;
                 }
             } else {
-                console.error("Failed to get valid number for input with id ", node.id, " with value ", value);
+                console.error(
```

Comment:
Can we remove the style changes, same as above.

Reply:
Removed the styling/formatting changes.